core_backend/app/configs/llm_prompts.py

```diff
@@ -89,21 +90,33 @@ def get_prompt(cls) -> str:
         [{",".join(cls._member_names_)}]
         """
 
+    @classmethod
+    def get_supported_languages(cls) -> list[str]:
+        """
+        Returns a list of supported languages.
+        """
+        return [lang for lang in cls._member_names_ if lang != "UNKNOWN"]
 
-# ----  Translation bot
 
-TRANSLATE_INPUT = """
+# ----  Translation bot
+TRANSLATE_FAILED_MESSAGE = "ERROR: CAN'T TRANSLATE"
+TRANSLATE_INPUT = f"""
     You are a high-performing translation bot for low-resourced African languages.
-    You support a maternal and child health chatbot.
+    You support a question-answering chatbot.
+    If you are unable to translate the user's input,
+    respond with {TRANSLATE_FAILED_MESSAGE}
     Translate the user's input to English from """
 
 # ---- Paraphrase question
-
-PARAPHRASE_INPUT = """
+PARAPHRASE_FAILED_MESSAGE = "ERROR: CAN'T PARAPHRASE"
+PARAPHRASE_INPUT = f"""
     You are a high-performing paraphrase bot.
-    You support a maternal and child health chatbot.
-    The user has asked a health question in English, paraphrase it to focus on
-    the health question. Be succinct and do not include any unnecessary information."""
+    You support a question-answering service.
+    The user has asked a question in English, paraphrase it to focus on
+    the question. Be succinct and do not include any unnecessary information.
+    Ignore any redacted and offensive words.
+    If no paraphrase is possible, respond with {PARAPHRASE_FAILED_MESSAGE}
```

Comment:
What do you mean by not possible? 

I was playing around with cases where paraphrasing seemed unnecessary, but 
 gpt is still giving paraphrased answers.. e.g. "What is Amul?" -> "What does Amul represent?". I guess it doesn't matter from the user's point of view 🤷 

Btw, I tried rephrasing the prompt but it did not help either:
```
    You are a high-performing paraphrase bot.
    You support a question-answering service.
    The user has asked a question in English. Paraphrase it to 1) focus on
    the question, 2) be succinct, 3) not include any unnecessary information,
    and 4) ignore any redacted and offensive words. If the original questions meet all
    of these conditions, don't paraphrase unnecessarily.
    If no paraphrase is possible, respond with {PARAPHRASE_FAILED_MESSAGE}
```

Reply:
Since it supports a question answering service, if the phrase is not a question, it doesn't know how rephrase it. Or if you give it a bunch on nonsensical or offensive stuff. This catches all those scenarios.